Foundation/Unit.swift

```diff
@@ -185,21 +221,45 @@ open class Dimension : Unit {
         self.converter = converter
         super.init(symbol: symbol)
     }
-    
+
+    public required init(symbol: String) {
+        fatalError("You must use the designated initializer.")
+    }
+
     open override func encode(with aCoder: NSCoder) {
         super.encode(with: aCoder)
         guard aCoder.allowsKeyedCoding else {
             preconditionFailure("Unkeyed coding is unsupported.")
         }
         aCoder.encode(self.converter, forKey:"converter")
     }
+
+    open override func isEqual(_ object: Any?) -> Bool {
+        guard let other = object as? Dimension else {
+            return false
+        }
+
+        if self === other {
+            return true
+        }
+
+        return super.isEqual(object) && self.converter == other.converter
+    }
 }
 
 open class UnitAcceleration : Dimension {
     
     /*
      Base unit - metersPerSecondSquared
      */
+
+    public required init(symbol: String) {
+        fatalError("init(symbol:) has not been implemented")
```

Comment:
... it would now be sufficient to simply override this with `super.init(symbol: symbol)` and the error message will still be readable.

Reply:
That's a very good suggestion. Thanks 👍